tests/gui/logger.py

```diff
@@ -332,13 +334,13 @@ def new_mousetool_event(self, tool_name, event_name, x, y, button):
 			# drage event
 			pass
 		else:
-			raise Exception("Event '%s' not supported." % event_name)
+			raise Exception("Event '{}' not supported.".format(event_name))
 
 		# Output debug information, no test code yet
 		if button:
-			log.debug("# %s.%s(%d, %d, '%s')" % (tool_name, event_name, x, y, button))
+			log.debug("# {}.{}({}, {}, '{}')".format(tool_name, event_name, x, y, button))
 		else:
-			log.debug("# %s.%s(%d, %d)" % (tool_name, event_name, x, y))
+			log.debug("# {}.{}({}, {})".format(tool_name, event_name, x, y))
```

Comment:
Other question this is not to easy to read:
`log.debug("# {}.{}({}, {})".format(tool_name, event_name, x, y))`
What do you think, is this maybe better?
`log.debug("# {tool}.{event}({:d}, {:d})".format(tool=tool_name, event=event_name, x, y))`